addons/project/models/project.py

```diff
@@ -516,6 +516,7 @@ def _read_group_stage_ids(self, stages, domain, order):
         string='Customer',
         default=lambda self: self._get_default_partner(),
         domain="['|', ('company_id', '=', False), ('company_id', '=', company_id)]")
+    commercial_partner_id = fields.Many2one('res.partner', related='partner_id.commercial_partner_id')
```

Comment:
Always show commercial partner on tasks: the commit looks good but we are too close to the release to add field, so sorry but can you drop it ? :s